integration-tests/src/test/java/oracle/weblogic/kubernetes/ItValidateWebhookReplicas.java

```diff
@@ -488,8 +488,8 @@ void testDomainReplicasTooHighTwoClustersNoReplica() {
     String expectedErrorMsg =
         "admission webhook \"weblogic.validating.webhook\" denied the request: Change request to domain resource '"
             + domainUid2
-            + "' cannot be honored because the replica count for cluster 'cluster-1' and 'cluster-2' "
-            + "would exceed the cluster size '5' when patching "
+            + "' cannot be honored because the replica count of each cluster in 'cluster-1, cluster-2' "
```

Comment:
Can you enable this test? It's still disabled.

Reply:
Hmm, I thought I enabled it. Will enable it after testing.